polaris.shopify.com/content/whats-new/version-11-color.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Version 11 Color
+description: Learn about what changes were introduced to color in Polaris v11.
+imageUrl: /images/updates/new-colors-cover.png
+keywords:
+  - color
+  - tokens
+---
+
+![Preview of new alias tokens](/images/updates/new-colors-cover.png)
+
+## Why are we making changes?
+
+Over the past year we have invested a lot of time and energy into creating a more robust and cohesive token system in Polaris. We have updated typography, spacing, shadows and breakpoints. All of this work has made it easier than ever to build with Polaris and has unlocked new levels of composability within the system itself. We knew we needed to bring this same thinking to our color tokens as well.
```

Comment:
This paragraph might not be needed. It's a lot of not specific information and the real value in "Why are we making these changes" is in the dot points below.

Reply:
Makes sense to me @heyjoethomas 👍